src/RemoteStorageService/index.js

```diff
@@ -0,0 +1,3 @@
+export * as Check from './Check';
```

Comment:
I like the way you built up the chain for these components. They're clearly related and encoding that relationship in their access (eg, `RemoteStorage.Warning.ForHoldings`) hammers it into the readers head.